### PR TITLE
Feat rating

### PR DESCRIPTION
**Pull request by junaidshaikh-js**
Added two types of rating
- Read-only rating 
- Live rating

Preview: https://deploy-preview-38--hydrogen-ui.netlify.app/docs/rating

### CODE REVIEW

File: components/rating.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  margin: auto 0.5rem;
+}
+
+.static-rating label {
+  cursor: auto;
+}
+
+.rating label:hover,
+.rating label:hover ~ label {
+  color: #ffb3009c;
```
Comment (Hamdrive) [inline components/rating.css:25]: Also prefer to store the color in a variable

**Review by Hamdrive — COMMENTED** [review]
Code written is quite good, lots of proper practices followed
Everything looks good, and code can be merged once suggested changes are implemented.

**Review by raghav6202 — APPROVED** [review]
Looks good to merge. Nice U.I👍

**Review by VLeads — APPROVED** [review]
Nice work. Go ahead !

File: components/rating.css (hunk 1 above)
Comment (junaidshaikh-js) [reply components/rating.css:25]: This color is used only once in hence not storage into the variable

**Pull request merged**